samples/bootloader/README.rst

```diff
@@ -80,6 +80,12 @@ If you choose to do so, use the Python scripts in ``scripts\bootloader`` to crea
       This will erase the whole chip before programming the new image.
       In |SES|, choose :guilabel:`Target` > :guilabel:`Connect J-Link` and then :guilabel:`Target` > :guilabel:`Erase All` to erase the whole chip.
 
+   .. note::
+      On some chips (for example, nRF9160 or nRF5340), the provisioned data is held in the OTP region in UICR.
```

Comment:
Should we create a task to make a script to perform this check?

Reply:
Done